fava_investor/__init__.py

```diff
@@ -44,3 +48,10 @@ def query_func(self, sql):
     def build_tlh_tables(self, begin=None, end=None):
         return libtlh.get_tables(self.query_func, self.config.get('tlh', {}))
 
+    # Balances
+    # -----------------------------------------------------------------------------------------------------------
+    def get_balances(self, interval: Interval):
+        interval_count = 3
+        accounts = AccountsConfig.from_dict(self.ledger, self.config.get('performance', []))
+        return interval_balances(self.ledger, accounts.value, interval,
```

Comment:
using favainvestorapi, you should be able to get this to read:
return interval_balances(accapi, self.config.get('performance', {})